### DISCUSSION TITLE
Does anyone have a docker container for the environment?

### DISCUSSION THREAD

**Discussion by xuanyaoming** [PegasusSimulator/PegasusSimulator#9] [Q&A]
I followed the installation guide and it doesn't work. Probably because the poor support of ROS2 in my system. It keeps saying "can't import rclpy".
Anyway, I pretty give up making isaac sim working with ROS2, and I just noticed that ROS2 isn't the only package needs to install. As setting up the environment is a very challenging task for an AI developer like me (who has very little experince in C compiling), does anyone have a docker container so that I could quickly test my program in the simulations?

**Comment by ramzibou0735** [comment]
issue #15 talked about having installed PegasusSimulator through docker using dev_camera branch (i don't know what he meant by that tho)

**Comment by vidneu** [comment]
Hey, this is probably too late but I have a working devcontainer setup for isaac and pegasus if that helps
